x/tx/signing/textual/coins.go

```diff
@@ -284,6 +284,12 @@ func FormatCoins(coins []*basev1beta1.Coin, metadata []*bankv1beta1.Metadata) (s
 		if err != nil {
 			return "", err
 		}
+
+		// If a coin contains a comma, return an error given that the output
+		// could be misinterpreted by the user as 2 different coins.
+		if strings.Contains(formatted[i], ",") {
+			return "", fmt.Errorf("coin %s contains a comma", formatted[i])
+		}
```

Comment:
Nit: curious if we can future proof this by using a regex instead, s.t. we fail if it doesn't match the regex expression.